backend/src/main/java/com/daedan/festabook/place/controller/PlaceController.java

```diff
@@ -27,6 +31,32 @@ public class PlaceController {
     private final PlaceService placeService;
     private final PlacePreviewService placePreviewService;
 
+    @PostMapping
+    @ResponseStatus(HttpStatus.CREATED)
+    @Operation(summary = "특정 조직의 세부 사항 없이 플레이스 저장")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "201", useReturnTypeSchema = true),
+    })
+    public PlaceResponse createPlace(
+            @Parameter(hidden = true) @OrganizationId Long organizationId,
+            @RequestBody PlaceRequest request
+    ) {
+        return placeService.createPlace(organizationId, request);
+    }
+
+    @PostMapping("/detail")
```

Comment:
```suggestion
    @PostMapping("{placeId}/detail")
```
 플레이스 Id 정보는 URI 에 담기는 게 더 자연스러워 보여요!